docs/src/rules/no-underscore-dangle.md

```diff
@@ -153,6 +156,38 @@ class Foo {
 }
 ```
 
+### allowInArrayDestructuring
+
+Examples of **incorrect** code for this rule with the `{ "allowInArrayDestructuring": false }` option:
```

Comment:
The merge conflict on this PR is likely because of the `correct`/`incorrect` annotations that were added https://github.com/eslint/eslint/commit/9bb24c1deb7ab6743080520d11aefe9c6b8b8f2d#diff-f7a016589f0145832d73bcd7281bd71dd75f59d858f55ff5568ebf6f1777f8a1. When you rebase, can you also add those annotations to the new sections? Those annotations give us the green check/red X on examples in the [rendered rule page](https://eslint.org/docs/latest/rules/no-underscore-dangle#rule-details).